crates/router/src/core/user.rs

```diff
@@ -1924,10 +1944,17 @@ pub async fn terminate_two_factor_auth(
         }
     }
 
-    let current_flow = domain::CurrentFlow::new(user_token, domain::SPTFlow::TOTP.into())?;
+    let current_flow = domain::CurrentFlow::new(user_token.clone(), domain::SPTFlow::TOTP.into())?;
     let next_flow = current_flow.next(user_from_db, &state).await?;
     let token = next_flow.get_token(&state).await?;
 
+    let _ = tfa_utils::delete_totp_attempts_from_redis(&state, &user_token.user_id)
+        .await
+        .map_err(|error| logger::error!(?error));
```

Comment:
Do we need to handle this error? 
```suggestion
        .map_err(|error| logger::error!(?error))?;
```